tests/nest_tests/resources/minimal_neuron.nestml

```diff
@@ -0,0 +1,3 @@
+model minimal_neuron:
```

Comment:
What does having this kind of a neuron mean?

Reply:
Added a documentation header to the model.